### PR TITLE
removes .DS_Store 

### PR DESCRIPTION
**Pull request by JonnycatMeow**

### CODE REVIEW

**Review by moxie-coder — APPROVED** [review]
👍

**Comment by BeastlyGabi** [comment]
image

**Review by ghost — APPROVED** [review]

**Comment by JonnycatMeow** [comment]
> image

damn

**Review by libauror-a — APPROVED** [review]
Bruh! Take this one, like really; Take this one. No more .DS_Store! No more files! No more .DS_Store files! No more attributes! You've been ignored!

File: .gitignore [hunk 1]
```diff
@@ -18,4 +18,6 @@ export/*
 !.vscode/settings.json
 !.vscode/tasks.json
 !.vscode/launch.json
-!.vscode/extensions.json
\ No newline at end of file
+!.vscode/extensions.json 
+
+.DS_Store # Removes .DS_Store for good
```
Comment (ThePlank) [inline .gitignore:23]: But where's desktop.ini?

Comment (JonnycatMeow) [reply .gitignore:23]: idk

**Review by charlesisfeline — APPROVED** [review]

**Comment by JonnycatMeow** [comment]
@ShadowMario please merge this pull request

**Comment by MemeHovy** [comment]
> @ShadowMario please merge this pull request

he has his notifications off

**Review by NexIsDumb — APPROVED** [review]

**Comment by rivalryxd** [comment]
ngl i have no idea whats .DS_Store is but it sounds useless

**Review by BL1Z33 — APPROVED** [review]

**Comment by MemeHovy** [comment]
> ngl i have no idea whats .DS_Store is but it sounds useless

DS_Store is a file that stores custom attributes of its containing folder, such as folder view options, icon positions, and other visual information

**Pull request closed without merging**